site/_shortcodes/EventCard.js

```diff
@@ -0,0 +1,275 @@
+    alt: event.title,
+  });
+
+  return html`
+    <enhanced-event-card>
+      <article id="${event.id}"
+               class="event-card pad-400 lg:pad-500 no-js"
+               tabindex="0"
+               show-details="">
+          <div class="display-flex">
+            <figure class="event-card__desktop-image gap-right-500 flex-shrink-none">
+              ${image}
+            </figure>
+
+            <div class="event-card__overview display-flex direction-column justify-content-between">
+              <div>
+                <div class="event-card__title">
+                  <figure class="event-card__image gap-right-200 flex-shrink-none">
+                    ${image}
+                  </figure>
+
+                  <a href="${event.externalUrl}"
+                     class="decoration-none"
+                     target="_blank">
+                    ${event.title}
+                  </a>
+                </div>
+
+                <p>
+                  ${event.summary}
+                </p>
+
+                <div class="event-card__meta display-flex gap-bottom-400">
+                  <p class="display-flex align-center gap-right-500">
+                      ${pinIcon}
+                      ${event.location}
+                  </p>
+
+                  <p class="display-flex align-center">
+                      ${calendarIcon}
+                      ${formatDateShort(event.date)}
+                  </p>
+                </div>
+              </div>
+
+              <button class="material-button button-filled button-round">
+                ${i18n('i18n.common.see_whos_joining')}
+              </a>
+            </div>
+          </div>
+
+          <div class="event-card__details gap-top-400 grid-cols-1 grid-gap-400 lg:grid-cols-2">
+              ${event.sessions
+                .filter(session => session.type === 'speaker')
+                .map(session =>
+                  EventSessionCard(html`
+                    <a
+                      href="${event.externalUrl}"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      class="event-card__title decoration-none gap-bottom-300 display-flex align-center justify-content-between"
+                    >
+                      <span class="display-flex align-center">
+                        ${Img({
+                          src: session.speaker.image,
+                          width: 40,
+                          height: 40,
+                          alt: i18n(session.speaker.title),
+                          class:
+                            'flex-shrink-none height-600 width-600 rounded-full gap-right-300',
+                        })}
+                        ${i18n(session.speaker.title)}
+                      </span>
+
+                      ${launchIcon}
+                    </a>
+
+                    <p class="event-card__sub-title gap-bottom-200">
+                      ${i18n('i18n.common.talk_title')}
```

Comment:
This should live in i18n.events not common - or is it reused somewhere? Same for other i18n strings